uikit/src/main/java/com/midtrans/sdk/uikit/views/banktransfer/status/BankTransferStatusPresenter.java

```diff
@@ -67,8 +77,15 @@ public String getVaExpiration() {
                     expiration = transactionResponse.getPermataExpiration();
                     break;
                 case PaymentType.ALL_VA:
-                    //expiration is based on other VA processor
-                    expiration = TextUtils.isEmpty(transactionResponse.getBniExpiration()) ? transactionResponse.getPermataExpiration() : transactionResponse.getBniExpiration();
+                    String bank = transactionResponse.getAccountNumbers().get(0).getBank();
+
+                    if (bank.equals(BRI)) {
```

Comment:
please extract logic into private method. please use switch-case and default

Reply:
resolved in the latest commit, please check.